mindsdb/integrations/handlers/pycaret_handler/requirements.txt

```diff
@@ -1,2 +1,2 @@
-pycaret
-pycaret[models]
+pycaret[models]==3.3.2
+scikit-learn==1.2.2
```

Comment:
Do we need scikit-learn? I think this is handled by `pycaret` https://github.com/pycaret/pycaret/blob/master/requirements.txt#L10

Reply:
Yeah we don't need it , it will be handled by pycaret , I just add it so that the user will check their existing scikit-learn version and compare (which can lead to initial issue).

Reply:
So the issue is related to wrong `scikit-learn` version right? If so, we need to make sure that one is compatible with the main requirements file version

Reply:
Yeah the latest version of pycaret doesn't work with latest versions of sckit-learn. so we have to delete any already installed latest versions of sckit-learn and then install pycaret or the version of scikit-learn specified

Reply:
Yeah @ZoranPandovski ?